client/src/components/Chat.jsx

```diff
@@ -258,6 +258,10 @@ const Chat = () => {
 	const showBadword = (id) => {
 		setBadwordChoices({ ...badwordChoices, [id]: 'show' });
 	};
+ 
+	function getRepliedMessage(replyTo){
+		return decryptedMessages.find(object=>object.id===replyTo)
+	}
```

Comment:
Pls run ESLint on this file or install ESLint VS Code extension

Reply:
I use Prettier to format the code in this file.